migrations/sql/V2018.09.17.1344__MoreCountryMetadata.sql

```diff
@@ -0,0 +1,99 @@
+-- NOTE for review:
+--
+-- does it makes sense to prefix these tables with country_ or not?
+-- check null ness for everything
+-- everything that needs to be an enum table done
+-- no "data" in column names (e.g., thing_2008, thing_yf, etc)
+-- some of the not-null constraints need to be relaxed until we have the data import done - this needs writing out as a second import.
+
+
+CREATE TABLE gavi_region (
```

Comment:
Can we get a little extra in here and in the corresponding montagu-data import for this: specifically what are the last two in

```
   id               name
1  AA  Anglophone Africa
2  FA Francophone Africa
3  SW         SEARO/WPRO
4 EEP     EMRO/EURO/PAHO
```

This is a little easier to get right now

Reply:
The last two are WHO regions. I have made them explicit in `gavi_region.csv`. Should be updated automatically.

Reply:
Ah, I see - the last two entries are 2 and 3 WHO regions each